plugin/src/main/resources/checkstyle.xml

```diff
@@ -181,6 +185,8 @@
     </module>
 
     <!-- Enable suppression comments -->
+    <module name="SuppressWarningsFilter" />
```

Comment:
The comment above this line should be moved since it is attached to the `SuppressionCommentFilter` configuration. If we want to comment on this line, it should probably read

```xml
<!-- Enable suppression annotations -->
```